CONTRIBUTING.md

```diff
@@ -530,6 +530,33 @@ Before continuing, please be sure to read the online documentation of pyscopg2 t
   - [Advanced parameter types](http://initd.org/psycopg/docs/usage.html#adaptation-of-python-values-to-sql-types)
 
 
+### Do not bypass the ORM
+
+You should never use the database cursor directly when the ORM can do the same thing! By doing so you are bypassing all the ORM features, possibly the transactions, access rights and so on.
+
+And chances are that you are also making the code harder to read and probably less secure (see also next guideline):
+
+```python
+# very very wrong
+cr.execute('select id from auction_lots where auction_id in (' +
+           ','.join(map(str, ids)) + ') and state=%s and obj_price>0',
+           ('draft',))
```

Comment:
@eLBati 
FYI I have created follow script to understand the real issue of sql injection:
https://github.com/vauxoo-dev/gist-vauxoo/blob/master/sql_injection.py